.cirrus.yml

```diff
@@ -134,7 +134,7 @@ freebsd_task:
 #   matrix:
 #     - name: Cirrus Linux AArch64 Clang
 #       arm_container:
-#         image: 'docker.io/rpcs3/rpcs3-ci-jammy-aarch64:1.2'
+#         image: 'docker.io/rpcs3/rpcs3-ci-jammy-aarch64:1.4'
```

Comment:
Why 1.4?

Reply:
Because the 1.5 aarch64 docker is not ready yet, it will be updated once it is and taken out of draft